scripts/calibration/jaiabot-calibration.py

```diff
@@ -171,11 +170,11 @@ def skipMotor():
     write("Y")
     write("J")
     rd()
-    print("current forward stop threshold is 1599 microseconds")
+    print("current forward stop threshold is 1575 microseconds")
```

Comment:
Should we be using magic numbers for these print outs? Is there a variable we can put here instead?